src/args.py

```diff
@@ -18,6 +18,7 @@ def create_arg_parser(env_required: bool = True,
     parser.add_argument("--included_options", default="", type=str)
     parser.add_argument("--seed", required=seed_required, type=int)
     parser.add_argument("--option_learner", type=str, default="no_learning")
+    parser.add_argument("--explorer", type=str, default="no_explore")
```

Comment:
does this make `CFG.interactive_action_strategy` removable?

Reply:
yeah, that will be part of the interactive approach refactoring